homeassistant/components/ai_speaker/__init__.py

```diff
@@ -1,46 +1,51 @@
-"""Support for AI-Speaker."""
-import logging
+"""The AI Speaker integration."""
+import asyncio
 
 from aisapi.ws import AisWebService
+import voluptuous as vol
 
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
 from homeassistant.helpers import aiohttp_client
 
 from .const import DOMAIN
 
-PLATFORMS = ["sensor", "media_player"]
-_LOGGER = logging.getLogger(__name__)
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
 
```

Comment:
We can remove the config schema. It's not used.
```suggestion
```

Reply:
done 5619c95